runtime/compiler/p/codegen/J9TreeEvaluator.cpp

```diff
@@ -7164,201 +7164,6 @@ static void genZeroInit(TR::CodeGenerator *cg, TR::Node *node, TR::Register *obj
    cg->stopUsingRegister(zeroReg);
    }
 
-TR::Register *outlinedHelperNewEvaluator(TR::Node *node, TR::CodeGenerator *cg)
```

Comment:
This function is the only use of the `TR_VariableLenArrayAlloc`, `TR_ConstLenArrayAlloc`, and `TR_ObjAlloc` per-CC snippets. Since they're no longer needed, all the code related to emitting them (`initializeCCPreLoadedArrayAlloc` and `initializeCCPreLoadedObjectAlloc` from `J9PPCSnippet.cpp` are in charge of doing that) and allocating space for them (see Power's implementation of `TR::getCCPreLoadedCodeSize`) should also be removed accordingly.

Reply:
Fixed!